rpaas/nginx/manager.go

```diff
@@ -0,0 +1,104 @@
+package nginx
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	defaultManagePort         = 8800
+	defaultPurgeTimeout       = time.Duration(1 * time.Second)
+	defaultPurgeLocation      = "/purge"
+	defaultPurgeLocationMatch = "^/purge/(.+)"
+	defaultVTSLocationMatch   = "/status"
+)
+
+type NginxManager struct {
+	managePort    uint16
+	purgeLocation string
+	client        http.Client
+}
+
+type NginxError struct {
+	Msg string
+}
+
+func (e NginxError) Error() string {
+	return e.Msg
+}
+
+func NewNginxManager() NginxManager {
+	return NginxManager{
+		managePort:    managePort(),
+		purgeLocation: purgeLocationMatch(),
+		client: http.Client{
+			Timeout: defaultPurgeTimeout,
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
```

Comment:
You can drop this line (according to line 92, the request is ever made at plain HTTP).

Reply:
Sure, I'll remove it!